api/proto/teleport/certificate/v1/ssh.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 Gravitational, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.certificate.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/certificate/v1;certificatev1";
+
+// SSH certificated marshaled as authorized key.
+message SSHCertificate {
```

Comment:
Is this specifically for SSH certificates and not other forms of public keys? Are we going to enforce that?